source/_data/glossary.yml

```diff
@@ -0,0 +1,17 @@
+- topic: Platform
+  description: A platform makes the connection to a specific software or hardware platform. The `pushbullet` platform works with the service from pushbullet.com.
+- topic: Component
```

Comment:
We should move `component` above `platform` to maintain the hierarchy. A component can have multiple platforms. Also, this may be a good place to clear up the confusion between platform and component. For example, Abode component provides the underlying core logic and infrastructure that can be used by individual platforms (e.g., Alarm, sensors, etc. )

Reply:
Based on the sort order, I think `component` will appear well above `platform`. Right now it appears to be sorted in alphabetical order by `topic`.

Reply:
`| sort: 'topic'` will do the sorting.

This is hopefully just a start. Collecting the details and provide an easy to understand description is the hard part. Nice formatting can be done later :wink: